components/infobox/commons/infobox_league.lua

```diff
@@ -251,21 +248,44 @@ function League:createInfobox()
 	local builtInfobox = self.infobox:widgetInjector(self:createWidgetInjector()):build(widgets)
 
 	if self:shouldStore(args) then
-		self.infobox:categories('Tournaments')
-		if not String.isEmpty(args.team_number) then
-			self.infobox:categories('Team Tournaments')
-		end
-		if String.isNotEmpty(args.player_number) or String.isNotEmpty(args.individual) then
-			self.infobox:categories('Individual Tournaments')
-		end
-		self.infobox:categories(unpack(self:getWikiCategories(args)))
+		self.infobox:categories(unpack(self:getCategories(args)))
 		self:_setLpdbData(args, links)
 		self:_setSeoTags(args)
 	end
 
 	return tostring(builtInfobox) .. WarningBox.displayAll(League.warnings)
 end
 
+--- Allows for overriding this functionality
```

Comment:
Isn't `getWikiCategories` what is supposed to be overriden?

Reply:
we will need it overwritable for #2604 anyways